tests/waveplate_stub_test.py

```diff
@@ -0,0 +1,108 @@
+from pnpq.devices.waveplate_stub import WaveplateStub
+from pnpq.errors import (
+    DeviceDisconnectedError,
+    WaveplateInvalidDegreeError,
+    WaveplateInvalidMotorChannelError,
+    WaveplateInvalidStepsError,
+)
+import pytest
+
+def test_access_without_connection():
+    wp = WaveplateStub()
+    with pytest.raises(DeviceDisconnectedError):
+        wp.identify()
+
+def test_connect():
+    wp = WaveplateStub()
+    wp.connect()
+    wp.identify()
+
+def test_home():
```

Comment:
Creating a WaveplateStub and calling `connect` in every test over and over again is really repetitive.

Instead, let's use a [test fixture](https://docs.pytest.org/en/8.0.x/how-to/fixtures.html). Let's create a fixture called `connected_waveplate` and pass that in instead.